components/Navbar.tsx

```diff
@@ -55,7 +55,7 @@ const Navbar: NextPage = () => {
               {navLinks.map((navLink) => (
                 <li key={navLink.path}>
                   {navLink.external ? (
-                    <a
+                    <Link
```

Comment:
add `aria-label="desktop navbar link"`

```suggestion
                    <Link aria-label="desktop navbar link"
```